IFC4x3/Properties/a/AvgDepthOfTreatment_2ATszB7MD5rOdgrmtnn2wi/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="AvgDepthOfTreatment_2ATszB7MD5rOdgrmtnn2wi" Name="AvgDepthOfTreatment" UniqueId="8a776f4b-1d63-45d5-89ea-d70df1c42eac" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
Propose renaming to "AvgTreatmentDepth".

Reply:
Fixed in [fa34e3f](https://github.com/bSI-InfraRoom/IFC-Specification/pull/676/commits/fa34e3f56f97c85fda90a91839411efbd6712470)